Pear/Profile/ProfilePromptTableViewCell.swift

```diff
@@ -74,18 +74,16 @@ class ProfilePromptTableViewCell: UITableViewCell {
 
     func configure(for user: MatchedUser, type: ProfileSectionType) {
         titleLabel.text = type.getTitle(for: user)
-//        let major = user.majors.count > 0 ? user.majors[0].name : ""
-        // TODO: Fix this in API response
-        let major = "Computer Science"
+        let major = user.majors.count > 0 ? user.majors[0].name : ""
```

Comment:
While `isEmpty` and `count == 0` are the same speed for arrays, I think using 'isEmpty' is more clear in general